twitter.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Project</title>
+    <link rel="stylesheet" href="twitter1.css" />
+  </head>
+  <body>
+    <!-- ... icon -->
+    <!-- <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 12h.01M12 12h.01M19 12h.01M6 12a1 1 0 11-2 0 1 1 0 012 0zm7 0a1 1 0 11-2 0 1 1 0 012 0zm7 0a1 1 0 11-2 0 1 1 0 012 0z" />
+</svg> -->
+    <div class="container">
+      <!-- LEFT MENU -->
+      <section class="left-menu">
+        <div class="lft-top--el-a">
+          <ion-icon name="logo-twitter" class="twitter"></ion-icon>
+          <div class="left-icons-n-list">
+            <div class="left-m-icons">
+              <ul class="list">
+                <li>
+                  <ion-icon name="home" class="house"></ion-icon>
+                </li>
+                <li>
+                  <p>
+                    <svg
```

Comment:
We can make our code a little easier to read if we import an SVG via the image tag, something similar to this `<img src="./images/my-svg.svg" />`